packages/samples/iife/www/atomic-react.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script crossorigin src="/cdn/deps/react/umd/react.production.min.js"></script>
+    <script crossorigin src="/cdn/deps/react-dom/umd/react-dom.production.min.js"></script>
+    <!-- TODO: Remove the need for `react-dom-server-legacy`, use `react-dom-server` instead. -->
+    <script crossorigin src="/cdn/deps/react-dom/umd/react-dom-server-legacy.browser.production.min.js"></script>
+
+    <script crossorigin src="/cdn/headless/headless.js"></script>
+    <script crossorigin type="module" src="/cdn/atomic/atomic.esm.js"></script>
+    <link rel="stylesheet" href="/cdn/atomic/themes/coveo.css" />
+    <script crossorigin src="/cdn/atomic-react/iife/atomic-react.min.js"></script>
+
+    <script crossorigin src="/cdn/deps/@babel/standalone/babel.min.js"></script>
+    <title>Atomic React test page</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="text/babel" data-plugins="transform-react-jsx">
```

Comment:
I decided to use babel here for two reasons.

1. It allows me to copy and paste the entire NextJS sample almost as-is, since it transforms JSX to vanilla JS.
2. It's what we were already recommending in our doc, and I think it's a good idea.